src/_usage/troubleshooting.md

```diff
@@ -0,0 +1,29 @@
+---
+permalink: /usage/troubleshooting
+title: Troubleshooting
+description: How to troubleshoot Quilt when there's an issue
+redirect-from:
+- /troubleshooting.html
+- /support/
+- /support.html
+- /help/
+- /help.html
```

Comment:
Please don't do this, redirects are only for redirecting people when pages have _moved_ - using them otherwise will lock those URLs out because browsers will cache the redirect, so we won't be able to use them for any other pages.

Reply:
Ah okay